avocado/utils/dmesg_utils.py

```diff
@@ -0,0 +1,114 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2018 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+
+"""
+Module for manipulate dmesg while running test.
+"""
+
+
+import time
+import os
+from . import process, genio
+
+_dmesg_log_file = '%s/dmesglog-%s' % (genio._log_file_dir,
+                                      time.strftime("%Y-%m-%d:%H:%M:%S"))
+
+
+class DmesgError(Exception):
+    """
+    Base Exception Class for all dmesg  utils exceptions
+    """
+    pass
+
+
+def clear_dmesg():
+    """
+    This function clear dmesg of system
+    and save older dmesg in a file
+    """
+
+    cmd = "dmesg -c"
+    out = process.system(cmd, timeout=30, ignore_status=True,
```

Comment:
the result of `process.system` is status and not output. It'd be better to call the variable `stat` or `status`.